web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -632,10 +469,6 @@
     "os.name": "System name",
     "prefix": "Prefix",
     "process":"process",
-    "processRecommendedOnHostChange": "Update metadata records",
```

Comment:
Some code cleanup related to these items seems missing:

- https://github.com/geonetwork/core-geonetwork/blob/f1a9cc38201712d8e23bf1e3a8471dfd0de52013/web-ui/src/main/resources/catalog/js/admin/SystemSettingsController.js#L414
- https://github.com/geonetwork/core-geonetwork/blob/f1a9cc38201712d8e23bf1e3a8471dfd0de52013/web-ui/src/main/resources/catalog/js/admin/SystemSettingsController.js#L423